src/main/java/com/dreammaster/coremod/transformers/recipenukers/GraviSuiteTransformer.java

```diff
@@ -0,0 +1,46 @@
+package com.dreammaster.coremod.transformers.recipenukers;
+
+import java.util.ListIterator;
+
+import org.objectweb.asm.Opcodes;
+import org.objectweb.asm.tree.AbstractInsnNode;
+import org.objectweb.asm.tree.ClassNode;
+import org.objectweb.asm.tree.InsnNode;
+import org.objectweb.asm.tree.MethodNode;
+import org.objectweb.asm.tree.TypeInsnNode;
+
+import com.dreammaster.coremod.DreamCoreMod;
+import com.dreammaster.coremod.IDreamTransformer;
+
+public class GraviSuiteTransformer implements IDreamTransformer {
+
+    @Override
+    public String[] targetedClasses() {
+        return new String[] { "gravisuite.GraviSuite" };
+    }
+
+    @Override
+    public void transform(ClassNode classNode) {
+        for (MethodNode mn : classNode.methods) {
+            if (mn.name.equals("afterModsLoaded")
```

Comment:
This won't cause conflict, the mixins don't have any `require` parameters therefore they won't cause any crash, they will just not apply

Reply:
I think this removal should be moved to GSNeo (and become part of https://github.com/GTNewHorizons/Gravitation-Suite-Neo/pull/21) to not split the logic into multiple mods.

Reply:
From what I understand gravisuite neo is used outside the pack, furthermore we greed that the ASM that removes recipes should be keep in the coremod because it is that same mod that adds them via the scripts.